client/src/components/settings/toggle-setting.css

```diff
@@ -80,50 +84,55 @@
 
 .toggle-radio-group {
   display: flex;
```

Comment:
I don't think we need flex on this any more. We aren't taking advantage of any of the other flex properties and removing it doesn't seem to have any advers effects. I think we can get rid of this entire ruleset.